app/src/main/java/com/pvp/app/ui/screen/calendar/CalendarViewModel.kt

```diff
@@ -60,10 +73,30 @@ class CalendarViewModel @Inject constructor(
                 .launchIn(viewModelScope)
         }
     }
+
+    suspend fun permissionsGranted(): Boolean {
+        val granted = client.permissionController.getGrantedPermissions()
+
+        return granted.containsAll(PERMISSIONS)
+    }
+
+    fun getDaysSteps(date: LocalDate) {
+        viewModelScope.launch {
+            val end = date.plusDays(1).atStartOfDay(ZoneId.systemDefault()).toInstant()
+            val start = date.atStartOfDay(ZoneId.systemDefault()).toInstant()
+
+            _stepsCount.value = healtConnectService.aggregateSteps(start, end)
+        }
+    }
 }
 
 data class CalendarState(
     val tasksMonth: List<Task> = listOf(),
     val tasksWeek: List<Task> = listOf(),
     val user: User? = null
+)
+
+val PERMISSIONS = setOf(
```

Comment:
I believe this should be lowercased, lint code suggestions by Android Studio should show it too. If they don't show it, means I am wrong and it should stay as it is.